feature-libs/checkout/occ/adapters/occ-checkout-payment.adapter.ts

```diff
@@ -219,8 +223,10 @@ export class OccCheckoutPaymentAdapter implements CheckoutPaymentAdapter {
     return values;
   }
 
-  private convertToMap(paramList: { key; value }[]) {
-    return paramList.reduce(function (result, item) {
+  private convertToMap(
+    paramList: { key: string; value: string }[]
+  ): { [key: string]: string } {
+    return paramList.reduce(function (result: { [key: string]: string }, item) {
```

Comment:
Just for sake off discussion, could the type of the value in the returned  map include `undefined`  like  `{ [key: string]: string | undefined }` ?  
I wonder if this remove the requirement to assign empty string as default values in `getParamsForPaymentProvider()`